CONTRIBUTING.md

```diff
@@ -51,6 +51,36 @@ Your contribution can be of numerous forms. We currently accept two ways to cont
 
 Since our [migration to GitHub] we gladly accept __[Pull Requests]__. Pull requests let you tell others about changes you have pushed to a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before the changes are merged into the repository. __Pull request is a prefered way to submit your work__ - it makes reviewing and merging your contribution much easier.
 
+#### Commit style
+
+When doing commits, use the following format:
+```
+[MODULE] Short summary of your commit
+
+Optional description that may include:
+- Lists like this one
+- JIRA bug ID
+- Coverity scan ID 
+- 'Quoted text'
+- etc.
+
+CORE-XXXX CIDXXXXX #pr-num
+```
+Note that you don't need a full stop in the summary. Also, prefer shorter summaries - 70 chars total - unless it can't be broken down. 
```

Comment:
One more note: `Always use upper-case characters for module name.`

I've saw PRs and commits that are not following this rule, so it seems not obvious enough.